### PR TITLE
adds support for validate command.

### PR DESCRIPTION
**Pull request by darwayne**
### Background
This adds a new validate command to PAT. Note this change depends on https://github.com/panther-labs/panther-enterprise/pull/13425

**Validation success**
image

**Validation Failure**
image

**New API endpoint not detected**
image

### Changes

* Adds new validate command
* Updates client to not validate requests against graphql schema
    * protected apis are not visible in the schema
### Testing

* Manual validation locally
* Unit tests

### CODE REVIEW

File: panther_analysis_tool/backend/public_api_client.py
Comment (dekatzenel) [inline panther_analysis_tool/backend/public_api_client.py:196]: I realize that async bulk upload is the same, but will this just go forever if the status never changes (e.g. there's an error in a way that prevents the status from changing)?

**Review by dekatzenel — COMMENTED** [review]
Generally LGTM, but we need to wait until rest of work is done to merge

File: panther_analysis_tool/backend/public_api_client.py
Comment (darwayne) [reply panther_analysis_tool/backend/public_api_client.py:196]: good call out I'll get a fix in for that

Comment (darwayne) [reply panther_analysis_tool/backend/public_api_client.py:198]: fixed

Comment (dekatzenel) [reply panther_analysis_tool/backend/public_api_client.py:198]: This is a good fix, but not what I was thinking of. I meant some sort of cap on retries, in case the status got stuck forever as NOT_PROCESSED

Comment (darwayne) [reply panther_analysis_tool/backend/public_api_client.py:198]: ooh, that's enforced on the backend api. currently set at 20minutes. If we want to change that behavior I think that falls out of the scope of this PR

File: panther_analysis_tool/backend/client.py
Comment (maxrichie5) [inline panther_analysis_tool/backend/client.py:36]:
```suggestion
class UnsupportedEndpointError(Exception):
```

**Review by maxrichie5 — DISMISSED** [review]

File: panther_analysis_tool/backend/client.py
Comment (darwayne) [reply panther_analysis_tool/backend/client.py:36]: nice catch let me fix that up

**Comment by dekatzenel** [comment]
@darwayne didn't we want this to wait for metrics?

**Comment by darwayne** [comment]
@dekatzenel yeah we can

**Review by dekatzenel — APPROVED** [review]

**Comment by darwayne** [comment]
We decided to not let this be blocked by the analytics work

**Pull request merged**
